src/cogs/jaegeraccounts.py

```diff
@@ -114,7 +115,8 @@ async def _get_accounts(self):
                                         hours=24)
                             break
                         except ValueError:
-                            raise InvalidSheetsValue("There seems to be a date (top row) or time (username) formatting error in the google sheets document.")
+                            raise InvalidSheetsValue(
+                                "There seems to be a date (top row) or time (username) formatting error in the google sheets document.")
```

Comment:
Looks like autoformatting, either on my old code or on your end.
```suggestion
                            raise InvalidSheetsValue("There seems to be a date (top row) or time (username) formatting error in the google sheets document.")
```